cli/replicaset/lua/cconfig/promote_election.lua

```diff
@@ -0,0 +1,21 @@
+
+local function promote(timeout)
+    local config = require('config'):get()
+    local fiber = require('fiber')
+
+    local election_mode = box.cfg.election_mode
+    local failover = config.replication.failover
+
+    if failover ~= 'election' then
+        error(('unexpected failover: %q, "election" expected'):format(failover))
+    end
+
+    if election_mode ~= 'candidate' and election_mode ~= 'manual' then
+        error(('unexpected election_mode: %q, "candidate" or "manual" expected'):format(election_mode))
+    end
+
+    box.ctl.promote()
+    box.ctl.wait_rw(timeout)
```

Comment:
```suggestion
    box.ctl.wait_rw(timeout)
```
Does it raise an error if the timeout expired?

Reply:
Yes, tested manually.

Reply:
Nice, thank you.